admin-dev/themes/new-theme/scss/pages/_orders_view.scss

```diff
@@ -0,0 +1,72 @@
+#order-view-page {
+  @media not print {
+    .card--details {
```

Comment:
```suggestion
    .card-details {
```

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/17617/commits/880b0ffdaca18f032d2cb941f37e0751b4745479